integration/common/tests/dbt/spark/thrift/result.json

```diff
@@ -605,7 +605,7 @@
         "eventType": "START",
         "inputs": [
             {
-                "facets": {
+                "inputFacets": {
```

Comment:
question: it looks like quite a big change, why has it change and should we communicate it somehow?

Reply:
If you track what OpenLineage spec defines, you might see that this is correct key for the `InputDatasetFacet`. Generally speaking we need some strategy what to do with changes that are actually fixes.

Reply:
>Generally speaking we need some strategy what to do with changes that are actually fixes.

If something was a bug, we fix it. This is different than something that wasn't the best solution, but it was intended as it.